x-pack/libbeat/cmd/inject.go

```diff
@@ -6,12 +6,17 @@ package cmd
 
 import (
 	"github.com/elastic/beats/libbeat/cmd"
+	"github.com/elastic/beats/libbeat/logp"
 
 	// register central management
+	"github.com/elastic/beats/x-pack/libbeat/licenser"
 	_ "github.com/elastic/beats/x-pack/libbeat/management"
 )
 
+var licenseDebugK = "license"
+
 // AddXPack extends the given root folder with XPack features
 func AddXPack(root *cmd.BeatsRootCmd, name string) {
+	licenser.Enforce(logp.NewLogger(licenseDebugK), licenser.BasicOrTrial)
```

Comment:
Probably just a naming issue, but I guess in addition to Basic or Trial, paying customer is also fine, right? :)

Reply:
Ah, I guess that means Basic or above, right?

Reply:
Yes, It's basic or above, I will rename the function for clarity.